src/LearningObjectStats/LearningObjectStatStore.ts

```diff
@@ -36,7 +50,32 @@ export class LearningObjectStatStore implements LearningObjectStatDatastore {
         unit: 0,
         course: 0,
       },
+      blooms_distribution: {
+        apply: 0,
+        evaluate: 0,
+        remember: 0,
+      },
     };
+
+    if (bloomsData && bloomsData.length) {
+      for (let i = 0; i < bloomsData.length; i++) {
+        switch (bloomsData[i]._id) {
+          case BLOOMS.APPLY:
+            stats.blooms_distribution.apply = bloomsData[i].value;
+            break;
+          case BLOOMS.EVALUATE:
+            stats.blooms_distribution.evaluate = bloomsData[i].value;
+            break;
+          case BLOOMS.REMEMBER:
+            stats.blooms_distribution.remember = bloomsData[i].value;
+            break;
+          default:
+        }
+      }
+    } else {
```

Comment:
```suggestion
    }
```
Non-error log should be removed